spec/rubocop/cop/style/one_line_conditional_spec.rb

```diff
@@ -108,4 +108,11 @@
     corrected = autocorrect_source('if 0 + 0 then 1 + 1 else 2 + 2 end')
     expect(corrected).to eq('0 + 0 ? 1 + 1 : 2 + 2')
   end
+
+  it 'does not break when one of the branches contains a retry keyword' do
+    expect_offense(<<-RUBY.strip_indent)
+      if true then retry else 7 end
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Favor the ternary operator (`?:`) over `if/then/else/end` constructs.
+    RUBY
```

Comment:
It seems that `retry` is surrounded by parentheses when using auto-correct.

```diff
-if true then retry else 7 end
+true ? (retry) : 7
```

I think that parentheses are unnecessary if possible :-)

Reply:
> I think that parentheses are unnecessary if possible :-)

Thanks for a great review! I will have a look. 🙂